### ISSUE TITLE
Thanks for the almost monthly smurf pool

### ISSUE THREAD

**Issue by WhitePearl5** [ValveSoftware/Dota2-Gameplay#15869]
### Description

That is, do I understand correctly, this is now the new reality of Dota 2: throwing players with a win rate of ~ 50% (or even lower) into the smurf pool, who dont play in MMR, or almost dont play? I'm just wondering, why do I get the smurf pool all the time? I play 50/50 (i mean, win-loose-win-loose) on my rating, everything goes fine for me, and then out of nowhere I get into the smurf pool, in which players in enemy team literally start squeezing me out of the game! Players who are not in the same party play together as pro-scene players, as well as people in my team (or are they not people at all?) without any communication in the chat, by voice or by clicking on the minimap, they gather at 4 and go to fight (most often just silently dying) and that's it.

Here's another example: for almost a week I couldn't get 2 points (0.5 each due to turbo games) in the win counter in Arcana on Rubick, because I just couldn't win because of the smurf pool!

I was sincerely waiting for the re-release of Arcana items on QoP, WR and ES in order to purchase them, but after the "Happy Life in the Smurf Pool" my patience just ran out - either fix my account so that I don't get into the smurf pool for nothing, or gift me a perma-ban so that I never play this game again. 

Just take a look at this series of defeats. Tell me, why do the system transfer me to a smurf pool?! 

Thank you for your attention, although it is unlikely that this topic will be noticed by someone who is connected with the game development team so that they can check the account statistics and find out what the problem is.

### Example Match ID (and possibly Timestamp)

You can open the statistics of my account yourself and see the history of the matches

### Screenshots

![1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/144199756/9b69b6fc-78a2-4a73-9191-2816aa262d47)
![2](https://github.com/ValveSoftware/Dota2-Gameplay/assets/144199756/41ff8d53-75e0-4a3b-8c71-27f2ffbd70b2)
![3](https://github.com/ValveSoftware/Dota2-Gameplay/assets/144199756/14b7efb9-ecd0-4edd-b02d-afbcaeb511ee)
![4](https://github.com/ValveSoftware/Dota2-Gameplay/assets/144199756/200d47ea-e857-4820-854d-75d96453e5fe)

**Comment by TobyFlendersonFromHR** [comment]
it seems like a low skill issue

**Comment by otvalil** [comment]
Dude you don't even know what smurf pool is, relax

**Comment by fsavariano** [comment]
No offense, but you don't have any evidence you are in a "smurf pool" at all. This is a site where community can actually communicate devs the bugs that annoy us. Don't ruin the tool by trying to convince people, either players or devs theirselves that it's the latter's fault you're on a losing streak.
Happens to all of us, you can always just vent about it with other players :)

**Comment by WhitePearl5** [comment]
> No offense, but you don't have any evidence you are in a "smurf pool" at all. This is a site where community can actually communicate devs the bugs that annoy us. Don't ruin the tool by trying to convince people, either players or devs themselves that it's the latter's fault you're on a losing streak. Happens to all of us, you can always just vent about it with other players :)
> 
> Edit: grammar

Indeed, one of the most problematic topics that absolutely all players, streamers and bloggers complain about is not a problem for developers, and it does not need to be raised) Let the developers continue to not care that the system is broken, and players with a negative win rate get even stronger players into the enemy team so that they completely lose the desire to play it. Cool! I like your way of thinking. It's strange that you're not one of the Dota developers yet.

Listen, maybe you shouldn't comment on such topics?

**Comment by HeatseekingBugSquisher** [comment]
IMO the only people who should be in smurf pool are actual smurfs and cheaters. NOBODY else should be in there. Griefers should go to low priority pool Single Draft Mode not smurf pool. Selling/Buying accounts and Cheating is waaaaaaay different to flaming like an asshole coz the game is frustrating. The two should definitely have different consquences. This sounds like Valve just being lazy AF by lumping all the undesirables together even though some of them have completely different circumstances.
